ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -898,6 +913,11 @@ impl Module {
         *self.var_names_computed.borrow_mut() = true;
     }
 
+    /// Return the size of a pointer (in bytes) for this module.
+    pub(crate) fn ptr_byte_size(&self) -> u8 {
```

Comment:
Pointers to different things could have a different size. I guess we can simplify that in our case. But if we simplify it, I don't know why different modules would have a different pointer size?

Reply:
Given that LLVM has already generated code with a given pointer size in mind, I think the *correct* thing to do would be to have the IR tell us the size of a pointer on a per-pointer basis. However, we've agreed to cut this corner for now and assume all pointers are `usize` long.

Reply:
Ideally we'd add assertions into ykllvm to check that this is always the case (e.g. in different address spaces). Then we can just reference that assertion in this file.

Reply:
I've had a go at that [here](https://github.com/ykjit/ykllvm/pull/118)

If merged, we would need to sync ykllvm to inherit the change.